polyfills/crypto.js

```diff
@@ -0,0 +1,234 @@
+// Polyfill for crypto and tinygo wasm
+if (!globalThis.crypto) {
+  globalThis.crypto = global.crypto = {
+    getRandomValues
+  };
+}
+
+// Below is the content of the npm package `polyfill-crypto.getrandomvalues`, and all of it's dependencies.
+// I'm including it in source because if it's include as a normal dep during the esbuild for esm, i.e. the
```

Comment:
This comment explains the presence of this file.  TL;DR; can't get the esm build to work without this in source.